pi-apps-terminal-bash-edition.sh

```diff
@@ -268,7 +258,15 @@ while [ "$1" != "" ]; do
 		;;
 		gui)
 			#open pi-apps regularly
-			$PI_APPS_DIR/gui
+			"$DIRECTORY/gui"
+			exit $?
+		;;
+		create-app)
+			"$DIRECTORY/createapp"
+			exit $?
+		;;
+		settings)
+			"$DIRECTORY/settings"
 			exit $?
 		;;
```

Comment:
same here.

Reply:
Removed in https://github.com/cycool29/PiApps-terminal_bash-edition/commit/04926f490147be8d829cdacfa23303ede15735c6.